schemas/parameters/schema/parameter.schema.json

```
@@ -48,6 +49,10 @@
         }
       ]
     },
+    "optional": {
+      "description": "Specify if this parameter if optional if the default value is null",
```

nitpick for wording and clarity
```suggestion
      "description": "Specify if this parameter is optional (only applicable if the default value is null)",
```